src/main/java/edu/tamu/cap/model/messaging/ContextAction.java

```diff
@@ -0,0 +1,7 @@
+package edu.tamu.cap.model.messaging;
+
+import edu.tamu.weaver.messaging.model.MessageActions;
+
+public enum ContextAction implements MessageActions {
+    METADATA_CREATE, METADATA_UPDATE, METADATA_DELETE, RESOURCE_CREATE, RESOURCE_DELETE
```

Comment:
These are rather Cap specific and maybe can be generalized. I understand the card specified these, but we should discuss a bit. Also, they should be in the weaver messaging module and not in each individual service.

Reply:
https://github.com/TAMULib/Cap/commit/24f0bf5609756bb3878700a9c0111742d75a7f69